content/ja/docs/reference/glossary/kops.md

```diff
@@ -1,25 +1,23 @@
 ---
-タイトル: kOps (Kubernetes オペレーション)
-id: kopsコップス
-日付: 2018-04-12
+title: kOps (Kubernetes Operations)
+id: kops
+date: 2018-04-12
 full_link: /docs/setup/production-environment/kops/
 短い説明: >
```

Comment:
you don't have to translate this one.
I fixed it on my PR.
https://github.com/kubernetes/website/pull/47963